compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -752,39 +763,37 @@ fn transcribe_metavar_expr<'a>(
 }
 
 /// Extracts an metavariable symbol that can be an identifier, a token tree or a literal.
-fn extract_var_symbol<'a>(
+fn extract_symbol_from_pnr<'a>(
     dcx: DiagCtxtHandle<'a>,
     ident: Ident,
```

Comment:
Should we only pass the span? It may be interesting to document that it's meant for error reporting only.

Reply:
Changed